src/components/__tests__/__snapshots__/radioGroup.test.tsx.snap

```diff
@@ -0,0 +1,163 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`set of radios { value: 'OFF' } 1`] = `
+.emotion-1 {
+  cursor: pointer;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  outline: none;
+  padding: 10px 0;
+  position: relative;
+  font-weight: 400;
+  color: #101721;
+  font-size: 14px;
+}
+
+.emotion-1[data-disabled='true'] {
+  cursor: not-allowed;
+  pointer-events: none;
+}
+
+.emotion-1[data-disabled='true'] .css-qatnln {
```

Comment:
Oh oh, weird.
This should have been an emotion class like `emotion-24`.
But that was not used so now it came out like this.
This might change if the hashing algo of emotion changes making out tests brittle.
Plus on top of that, this style will never target anything which is not tree shakeable.
Will look into this.